.ci/pipeline_integration_test

```diff
@@ -0,0 +1,175 @@
+#pwd= /tmp/build/a94a8fe5/pull-request-gardener.machine-controller-manager-provider-azure-pr.master/tmp/src/github.com/gardener/machine-controller-manager-provider-azure
+#source path= /tmp/build/a94a8fe5/pull-request-gardener.machine-controller-manager-provider-azure-pr.master
+#logs_path= /tmp/build/a94a8fe5/pull-request-gardener.machine-controller-manager-provider-azure-pr.master/.ci/controllers-test/logs
+
```

Comment:
we could remove these comments like you said

Reply:
Yes, sure.